geode-junit/src/main/java/org/apache/geode/test/micrometer/AbstractMeterAssert.java

```diff
@@ -98,6 +108,16 @@ public A hasTag(String key, String expectedValue) {
     return myself;
   }
 
+  public void hasNoTag(String key) {
+    isNotNull();
+    Meter.Id actualId = actualId();
+    if (actualId.getTag(key) != null) {
+      failWithMessage("Expected meter to have no tag with key <%s>"
```

Comment:
Are you breaking up strings to try to make spotless happy? This doesn't really seem worth it to me.

Reply:
I think that was spotless breaking up strings to make spotless happy. I'll join the lines.